corehq/sql_db/operations.py

```diff
@@ -1,30 +1,25 @@
-from __future__ import absolute_import
-from __future__ import unicode_literals
+from __future__ import absolute_import, unicode_literals
 
 import os
 import re
 from io import open
 
-import attr
 from django.conf import settings
 from django.db import connection, router
 from django.db.migrations import RunPython, RunSQL
 from django.template import engines
 
+import attr
+
+from corehq.util.django_migrations import noop_migration
+
 NOOP = object()
 
 
 class IndexRenameOperationException(Exception):
     pass
 
 
-def noop_migration():
-    """
-    A migration that does nothing. Used to replace old migrations that are no longer required e.g moved.
-    """
-    return RunPython(RunPython.noop, RunPython.noop)
```

Comment:
Why was this moved to `corehq.util.django_migrations`? It created a lot of noise in this PR.

Reply:
The concept of a noop migration isn't specific to SQL db operations - most of the places I added it in this PR weren't related to SQL models at all.  It seemed like the new location made more sense, and I wanted it to be right next to `skip_on_fresh_install`, since they're closely related (I imagine most usages of `skip_on_fresh_install` will eventually be converted to `noop_migration()`s.

I did briely consider importing it to `operations.py` so I wouldn't have to update the import statements, but figured that'd just be laziness.  I committed the move separately to hopefully make the noise ignorable, at least.